core/meta/src/TClass.cxx

```diff
@@ -3047,13 +3047,35 @@ TClass *TClass::GetClass(const char *name, Bool_t load, Bool_t silent, size_t hi
    }
 
    std::string normalizedName;
-   Bool_t checkTable = kFALSE;
 
+   // Check if this is a fundamental type already in the typesystem to delay
+   // as much as possible any lookup
    if (!cl) {
       {
          TInterpreter::SuspendAutoLoadingRAII autoloadOff(gInterpreter);
          TClassEdit::GetNormalizedName(normalizedName, name);
       }
+      std::string name_no_std = name;
+      TClassEdit::RemoveStd(name_no_std);
```

Comment:
What is the advantage of using this `name_no_std` rather than the `normalizedName`?